documentation/modules/post/networking/gather/enum_juniper.md

```diff
@@ -87,7 +87,7 @@ host         origin       service            public          private
 [*] Command shell session 2 opened (192.168.1.6:45623 -> 192.168.1.5:22) at 2018-02-19 21:32:20 -0500
 [*] Scanned 1 of 1 hosts (100% complete)
 [*] Auxiliary module execution completed
-resource (juniper_ex2200.rc)> use post/juniper/gather/enum_juniper
+resource (juniper_ex2200.rc)> use post/networking/gather/enum_juniper
```

Comment:
Why is this prompt `resource (juniper_ex2200.rc)` when we never loaded a resource file as far as the documentation is concerned? Same for other prompts in this module.